### PR TITLE
f-loyalty@v0.5.0

### PR DESCRIPTION
**Pull request by matthewhardern**
PR Adds the unauthenticated state for the loyalty main tab. I have added in migrated unit tests as part of its migration from HomeWeb.  

Screenshot 2021-09-08 at 14 42 43
Screenshot 2021-09-08 at 14 43 31

- [x] Unit tests have been [created|updated]

## Browsers Tested

- [x] Chrome (latest)

### CODE REVIEW

File: packages/components/organisms/f-loyalty/src/components/Unauthenticated.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<template>
+    <div :class="$style['c-loyalty-unauthenticated']">
+        <f-card
+            :class="$style['c-loyalty-unauthenticated-wrapper']"
+            has-outline>
+            <img
+                :class="$style['c-loyalty-unauthenticated-img']"
+                :src="`https://just-eat-prod-eu-res.cloudinary.com/image/upload/v1630068495/Experiments/Homeweb-Coreweb/stampcards-complex-object-login-promo-full-colour-${$t('percentage')}.svg `"
```
Comment (minrwhite) [inline packages/components/organisms/f-loyalty/src/components/Unauthenticated.vue:8]: Looking at the dish images loaded [here](https://www.just-eat.co.uk/restaurants-mcdonalds-bristol/menu) vs [here](https://www.menulog.com.au/restaurants-mcdonalds-peninsula-fair/menu) we should be using different cloudinary domains depending on the target market, e.g. `just-eat-prod-sg-res.cloudinary.com` for AU. Something to bring up with the content team I think, and can then add the correct domain into the localization config.

File: packages/components/organisms/f-loyalty/src/tenants/en-GB.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ const messages = {
             title: 'How it works'
         }
     },
+    unauthenticated: {
+        loginTitle: 'Log in to get started',
+        loginButtonText: 'Log in'
+    },
```
Comment (minrwhite) [inline packages/components/organisms/f-loyalty/src/tenants/en-GB.js:24]: Please can we add this to AU as well

**Review by minrwhite — COMMENTED** [review]
Couple of points - additionally could we also look at adding visual tests (as per [f-checkout example here](https://github.com/matthewhardern/fozzie-components/tree/f-loyalty/unauthenticated/packages/components/organisms/f-checkout/test/visual))to check that all changes in appearance as the component evolves are as expected

File: packages/components/organisms/f-loyalty/src/components/Unauthenticated.vue (hunk 1 above)
Comment (matthewhardern) [reply packages/components/organisms/f-loyalty/src/components/Unauthenticated.vue:8]: As talked about elsewhere I will talk to the content team about this.

**Comment by matthewhardern** [comment]
> Couple of points - additionally could we also look at adding visual tests (as per [f-checkout example here](https://github.com/matthewhardern/fozzie-components/tree/f-loyalty/unauthenticated/packages/components/organisms/f-checkout/test/visual))to check that all changes in appearance as the component evolves are as expected

As I have only just got access to percy and browserstack I will create a later PR to add in the visual tests.

**Review by minrwhite — DISMISSED** [review]

**Review by minrwhite — APPROVED** [review]

**Review by AlixRichards — APPROVED** [review]

**Pull request merged**